react-native/training/src/app/screens/TodoList/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import actionCreators from '../../../redux/todoList/actions';
+
+import TodoList from './layout';
+
+function TodoListContainer({ items, addItem, removeItem, toggleItemCompleted, removeCompleted }) {
+  return (
+    <TodoList
+      items={items}
+      addItem={addItem}
+      removeItem={removeItem}
+      toggleItemCompleted={toggleItemCompleted}
+      removeCompleted={removeCompleted}
+    />
+  );
+}
+
+TodoListContainer.propTypes = {
+  items: PropTypes.arrayOf(
```

Comment:
I think that this prop you can move to `constants` file because you're use this prop here and in another files